types/tests/webElement.test-d.ts

```diff
@@ -147,6 +147,7 @@ describe('new element() api', function () {
     expectType<ElementValue<string>>(elem.getAccessibleName());
     expectType<ElementValue<string>>(elem.getAriaRole());
     expectType<ElementValue<string>>(elem.getCssProperty('height'));
+    expectType<ElementValue<string>>(elem.takeScreenshot());
```

Comment:
I am curious if the below tests pass or fail
```javascript
    expectType<ElementValue<string>>(elem.takeScreenshot(true));
```

Reply:
It will fail now.

Do you want me to add negatives as well? There are a lot of commands so I am just checking if the expected syntax works or not right now. Anyways, the negatives for many would not even make sense, like `getText()` also does not take any argument, so what should we put as an argument for a negative?

Reply:
I don't believe so. I was simply verifying whether or not TSD detects the error.